### ISSUE TITLE
Fully automated tests

### ISSUE THREAD

**Issue by hoegrammer** [hoegrammer/democracy#31]
I.e. run on save, run on travis

**Comment by Shakyamuni177te** [comment]
Hi Naomi,

Is the project still going?

If so I can begin to work on this issue.

All the best,
Peter David Carter
Director
Sassafras Associated LTD

**Comment by hoegrammer** [comment]
Do you mean do i still want help with it? Maybe but it does seem to have
lost momentum a bit. Maybe we should meet up and discuss where to go from
here

**Comment by hoegrammer** [comment]
still want _to_ help, i meant to say

**Comment by Shakyamuni177te** [comment]
Yeah, Aaron hasn't done anything for over three months, and that was just
a bit of HTML code. Meeting up would be very useful. Am back in Sheffield
for a job interview soon. I'll give you a call when I am :)

Peter
